### DISCUSSION TITLE
What are Live Photos?

### DISCUSSION THREAD

**Discussion by andrewbanchich** [ente-io/ente#3069] [Q&A]
When uploading photos, looking at the status of the uploads I see "The photo and video files from your Live Photos have been merged into a single file".

What does this mean? I don't have any folder on my computer called "Live Photos", and I can't find any Ente feature called that either. I also can't guess what this is even supposed to mean. It's under the "Uploads in progress" section.

Doing a search for this term, it seems like something iPhone specific?

**Comment by mnvr** [comment]
Live Photos is indeed iPhone specific, but subsequently Android also added support for a similar mechanism called Motion Photos. Ente treats both of these as a "Live Photo".

If unrelated photos and videos are being clubbed together in the desktop app as a live photo, you could try the [nightly build](https://github.com/ente-io/photos-desktop/releases) or wait for the next release, because since the last release we've added a threshold that the photo and video being clubbed together should be within 2 minutes of each other (to reduce false positives).

**Reply by andrewbanchich** [reply]
1. I'm a bit unclear about this feature - I thought Ente said it doesn't modify photos at all and they will be exactly as they were when they had been uploaded. But the message "have been merged into a single file" implies this is not the case.
2. If Ente does have some process for finding similar photos and merging them into a single file, could that be updated to be an opt-in feature we select in the upload process?
3. General feedback - it would be nice if the Live Photos message were a link to some Ente docs on this to explain what they are and what's happening during the process.
